Framework/DataHandling/src/SaveRKH.cpp

```diff
@@ -209,7 +209,7 @@ void SaveRKH::write2D() {
   bool requireNewLine = false;
   int itemCount(0);
   for (size_t i = 0; i < ySize; ++i) {
-    const auto &ydata = m_workspace->readY(i);
+    const auto &ydata = m_workspace->y(i);
     for (size_t j = 0; j < xSize; ++j) {
```

Comment:
The naming `xSize` and `ySize` is quite confusing/ambiguous in this context. Also, you could just use a range-based loop since `j` is only used for indexing the ydata. e.g `for(const auto yVal: m_workspace->y(i)){...}`